shuup/front/template_helpers/general.py

```diff
@@ -307,13 +308,7 @@ def _get_page_range(current_page, num_pages, range_gap=5):
 
 @contextfunction
 def get_shop_language_choices(context):
```

Comment:
The naming is strange as this uses `settings` as a source for languages and not `shop` (`request.shop`). I can see this is not something you added just now, but should we consider changing this? `get_default_language_choices` and when we have mechanics in place for shop languages, add that separately?